packages/react-scripts/layout/views/layout.ejs

```diff
@@ -30,6 +30,13 @@
     <% if (process.env.NODE_ENV === 'production') { %>
       <% include partials/dynatrace %>
     <% } %>
+
+    <% if (feature('frontier_snow_unsupportedBrowser')) { %>
```

Comment:
`feature` looks up XPRMNTL experiments, not feature flags. You would need to use `getFeatureFlag()` instead for a split.io feature flag